Emulator/PAK.Stream.Emulator/Pak/PakBuilder.cs

```diff
@@ -184,6 +184,7 @@ public unsafe MultiStream Build(IntPtr handle, string filepath, Logger? logger =
                     if (format == FormatVersion.Version1)
                         length = (int)Align(length, 64);
 
+                    // TODO Fix failed to read all reqeuested bytes error coming from here (entry stream's length is wrong maybe?)
```

Comment:
Was this related to 79148f91a3d2c7e59bfd0f1ddd95060cd7b247be ?

Reply:
Yeah, I think it was. 
Either way, I haven't seen the issue at all in quite a while so I'm fairly sure it is fixed so I removed those comments (hopefully I'm not proven wrong)